cmd/server/assets/realmadmin/_form_security.html

```diff
@@ -123,4 +140,19 @@
   </div>
 </form>
 
+<script type="text/javascript">
+  $(function(){
+    let $grace = $('#mfa-grace-period');
+    let $mfaMode = $('#mfa-mode');
+
+    $mfaMode.change(function() {
+      if ($mfaMode.val() == 1) {
+        $grace.prop("disabled", false);
```

Comment:
I think I'd prefer to hide this option entirely instead of just toggling it's state.